test_runner/regress/test_tenants.py

```diff
@@ -475,3 +476,37 @@ def only_int(samples: List[Sample]) -> int:
     assert counts
     log.info(f"directory counts: {counts}")
     assert counts[2] > COUNT_AT_LEAST_EXPECTED
+
+
+def test_tenants_parallel_creation(neon_simple_env: NeonEnv):
+    """
+    (Relaxed) regression test for issue that led to https://github.com/neondatabase/neon/pull/9268
+    Create many tenants in parallel and then restart endpoints
+    """
+    env = neon_simple_env
+    pageserver_http_client = env.pageserver.http_client()
+
+    # This param needs to be 200 to reproduce the limit issue
+    n_threads = 8
+    barrier = threading.Barrier(n_threads)
+
+    def test_tenant():
+        # Create new tenant
```

Comment:
Could this instead be a pythonic doc comment (like your test case has with a string) and describe what is the aim of this fn? It is very much redundant to document calling a method called `create_tenant` as `Create new tenant`.

Reply:
removed, I still see value in such comments but whatever